presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/ValidateDependenciesChecker.java

```diff
@@ -257,13 +255,7 @@ public Void visitFilter(FilterNode node, Set<VariableReferenceExpression> boundV
             // Only verify names here as filter expression would contain type cast, which will be translated to an non-existent variable in
             // SqlToRowExpressionTranslator
             // TODO https://github.com/prestodb/presto/issues/12892
```

Comment:
Can we change this? I'm pretty sure this issue has been fixed for a long time (see https://github.com/prestodb/presto/issues/12891, https://github.com/prestodb/presto/issues/13116) and I would guess wouldn't be relevant anyway since we no longer have expressions in the plan.